superset-frontend/src/components/Menu/LanguagePicker.tsx

```diff
@@ -16,10 +16,12 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React, { useState } from 'react';
-import { Select } from 'src/common/components';
-import { styled, useTheme } from '@superset-ui/core';
-import Icons from 'src/components/Icons';
```

Comment:
Is there a reason why you are reverting back to the older Icon implementation?

Reply:
Yes. All menus are using the old implementation. If I change to the new one just here, we have differences in icons sizes. When we migrate the menu icons, then we should also migrate the icon in `LanguagePicker`.